README.md

```diff
@@ -0,0 +1,34 @@
+# Bootstrap Design Replication Project
+
+## Overview
+
+This project is a continuation of learning and strengthening my Front-end skills, improving code quality and better understanding of project workflows. Here I will replicate a chosen design from Boostrap as closely as possible using **HTML**, **SASS** and **Javascript** followed by **BEM** methodology.
+
+## Purpose
+I am building this project in order to deepen my understanding of:
+* **DRY**: Writing clean and reusable code
+* **Progressive Enhancement**: Building the project's solid, basic structure and progressively adding layers of styles and interactivity
+* **Mobile-First Apporach**: Designing layouts for smaller screens first and then enhancing them for larger resolutions
+* **Architecture**: I want to learn how to build and structure the project from scratch, without relying on tools like Webpack or Vite
+* **Rebase Flow**: In previous projects I used a merge-based workflow, so in this project I aim to learn and adopt rebase approach in order to improve my **Git** proficiency
+
+## Project Structure
+The project structure includes the following directories and files:
+```
+/dist           # Distibution folder for the final build
+/src            # Source files (.html, .scss, assets, etc.)
+README          # Project documentation
+build.sh        # Bash script for building the project into dist directory
+.gitignore      # Git ignore file
+```
+Although this project does not utilize Webpack, Vite or similar bundlers, I will write a custom **bash script** (build.sh) to transpile the application into the `/dist` folder. This approach will help me better understand project build proceesses and workflows.
+
+## Design Proposal
+Below are four Bootstrap designs I am considering for this project. Please let me know in the PR comment which design you would prefer me to implement:
+1. [FreshCart](https://freshcart-template.codescandy.com/)
+2. [Coach.](https://codescandy.com/coach/bootstrap-5/landing-preview.html)
+3. [Finder](https://finder-react.createx.studio/)
+4. [Pipeline](https://pipeline.mediumra.re/index.html)
```

Comment:
IMO all the proposals are at a similar level of complexity, but the last one is the most interesting due to its more advanced menu.